ufl/finiteelement/elementlist.py

```diff
@@ -259,6 +259,13 @@ def show_elements():
 register_element("Gauss-Lobatto-Legendre Edge L2", "GLL-Edge L2", 0, L2, "L2 Piola", (0, None),
                  ("interval",))
 
+# directly-defined serendipity elements ala Arbogast
+# currently the theory is only really worked out for quads.
+register_element2("Sdirect", 0, H1, "physical", (1, None), ("quadrilateral",))
+
+register_element2("Sdirect_Hdiv_full", 1, HDiv, "physical", (1, None), ("quadrilateral",))
+register_element2("Sdirect_Hdiv_reduced", 1, HDiv, "physical", (1, None), ("quadrilateral",))
```

Comment:
These names look odd. Normally, one uses spaces in long UFL element names, not underscores.  I think they should look more like titles than variable names.

Reply:
c.f. `"Nedelec 1st kind H(div)"`, `"Extended-Gauss-Legendre Edge L2"`, or `"Q- Lambda L2"`